packages/terra-show-hide/src/terra-dev-site/test/show-hide/StatesShowHide.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ShowHide from '../../../ShowHide';
+
+const StatesShowHideTest = () => (
```

Comment:
is this meant to be stateless show hide test?

Reply:
This example is meant to test styling. I was copied the name from StatesHyperLink.text.jsx. Which name would make more sense? Something like StylingShowHide.test.jsx?